src/Symfony/Component/DependencyInjection/Compiler/ServiceReferenceGraph.php

```diff
@@ -99,15 +83,7 @@ public function connect($sourceId, $sourceValue, $destId, $destValue = null, $re
         $destNode->addInEdge($edge);
     }
 
-    /**
-     * Creates a graph node.
-     *
-     * @param string $id
-     * @param string $value
-     *
-     * @return ServiceReferenceGraphNode
-     */
-    private function createNode($id, $value)
+    private function createNode(string $id, $value): ServiceReferenceGraphNode
```

Comment:
Missing `string $value` AFAIK.

Reply:
The correct type is "mixed", I'll submit a PR on lower branches to fix it.